phpcs.xml.dist

```diff
@@ -57,4 +58,8 @@
     <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod">
         <exclude-pattern>lib/Doctrine/ORM/Mapping/*</exclude-pattern>
     </rule>
+
+    <rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName">
+        <exclude-pattern>lib/Doctrine/ORM/Annotation/*</exclude-pattern>
```

Comment:
This looks fishy. Worth inspecting in a second PR just doing this one /cc @carusogabriel

Reply:
This is because imho doctrine/annotations doesn't properly support unqualified names in annotations, at least I repeatedly ran into test failures so I explicitly disabled it until further investigation.